sdks/wasm/src/library_mono.js

```diff
@@ -562,7 +572,18 @@ var MonoSupportLib = {
 		});
 	},
 
-	mono_wasm_add_obj_var: function(className, objectId) {
+	mono_wasm_add_getter_var: function(className) {
+		var value = `${Module.UTF8ToString (className)} { get; }`;
+		MONO.var_info.push({
+			value: {
+				type: "symbol",
+				value: value,
+				description: value
+			}
+		});
+	},
+
+	mono_wasm_add_obj_var: function(className, toString, objectId) {
```

Comment:
let's consolidate several of these similar methods in a follow up by passing the subtype as another argument or as part of the object id